code/cli/munki/shared/utils/scriptutils.swift

```diff
@@ -31,7 +31,7 @@ class ScriptRunner: AsyncProcessRunner {
     var combinedOutput = ""
 
     func linesAndRemainderOf(_ str: String) -> ([String], String) {
-        var lines = str.components(separatedBy: "\n")
+        var lines = str.split(whereSeparator: \.isNewline).map(String.init)
```

Comment:
This eliminates all empty strings, though:
"Hello\n\nGoodbye\n"
is split into ["Hello", "Goodbye"]
instead of  ["Hello, "", "Goodbye"]

Reply:
Looks like omittingEmptySubsequences is the ticket:

    var lines = str.trailingNewlineTrimmed.split(omittingEmptySubsequences: false, whereSeparator: \.isNewline).map(String.init)